objerve.py

```diff
@@ -29,7 +29,17 @@ def print_stack(self):
         print(*traceback.format_list([summary]))
 
 
-def watch(cls):
-    for key, val in cls.watch_dict.items():
-        setattr(cls, key, Descriptor(key, val))
-    return cls
+def watch(**kwargs):
```

Comment:
Currently nothing happens when an unrecognised kwarg key is aded. I don't necessarily think that adding validation is required, however it may be worth alerting the user of the correct keys?

Reply:
We can deal with the verification later.  It looks seems harmless for now.

Reply:
We can deal with the verification later. It seems harmless for now.